resources/assets/js/components/JobBuilderSkills/JobBuilderSkills.tsx

```diff
@@ -1,5 +1,11 @@
 /* eslint-disable camelcase, @typescript-eslint/camelcase */
-import React, { useState, useRef, useReducer, useEffect } from "react";
+import React, {
+  useState,
+  useRef,
+  useReducer,
+  useEffect,
+  useCallback,
```

Comment:
Looks like you're getting _hooked_ on hooks 🙂

Reply:
Haha Eslint was complaining